### ISSUE TITLE
Bits in missing/v2 depend on missingo/v1

### ISSUE THREAD

**Issue by yarikk** [anacrolix/torrent#355]
Having multi-version internal dependencies, along spoiling the style, breaks vendoring.

The library should be made consistent on which version of sub-packages they depend on.

**Comment by anacrolix** [comment]
It's not clean, but are you sure it breaks vendoring? `go mod vendor` seems to work as intended.

**Comment by yarikk** [comment]
You're right, I should be more specific: for various reasons we are locked away from the usual _go_ tool, so it breaks vendoring for us. What I'm saying is that the status quo has impact beyond stylistics for some users, and having this addressed would be so, too.
